authui/src/authflowv2/customSelect.ts

```diff
@@ -26,6 +27,14 @@ export class CustomSelectController extends Controller {
     initialValue: String,
   };
 
+  get bodyScrollLock() {
+    const ctr = this.application.getControllerForElementAndIdentifier(
+      this.element,
+      "body-scroll-lock"
+    );
+    return ctr as BodyScrollLockController | null;
+  }
+
```

Comment:
Follow this guide to coordinate these two controllers. https://stimulus.hotwired.dev/reference/controllers#cross-controller-coordination-with-events

Ultimately, it should look like

```
<div data-controller="custom-select body-scroll-lock"
  data-action="custom-select:open->body-scroll-lock#lock custom-select:close->body-scroll-lock#unlock">
```